Automattic/vip-design-system
# Deprecate old tabs

## Description

This PR removes the old tabs in favor of the new Accessible ones. 
There is no code changes aside from file and component renaming (since we're using the `Tabs.js` file it shows as changed, but we're just using the `NewTabs.js` content in it)

## Checklist


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Pull down PR.
1. `npm run dev`.
1. Open [Storybook.](http://localhost:6006/?path=/story/tabs--default)
1. Verify that the tabs work.
